src/Aspire.Dashboard/Components/Pages/Metrics.razor

```diff
@@ -57,7 +57,16 @@
                 <FluentIcon slot="end" Icon="Icons.Regular.Size20.Clock" Style="margin-right:5px;"/>
             }
 
+            @* the label and select do not have margins applied to them due to have slot="end", so
+               we can just add right margin equal to the gap applied by the toolbar * 2 to the label *@
+            <FluentInputLabel slot="end"
+                              Label="@Loc[nameof(Dashboard.Resources.Metrics.MetricsSelectADuration)]"
+                              ForId="@_selectDurationId"
+                              Orientation="Orientation.Horizontal"
+                              style="margin-right: calc(var(--toolbar-item-gap) * 2)" />
```

Comment:
On the structured logs page a divider is used to space them out: https://github.com/dotnet/aspire/blob/37d57597602ff423d82bc38ef99bbbf4d1c0f197/src/Aspire.Dashboard/Components/Pages/StructuredLogs.razor#L57

I think we should do that to be consistent.

Also, I see the structured logs page has a simple span instead of a label. Could you update that page to use a label as well?